CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Increse requested time limits in GCHP integration tests (compile 2h30m, run 5h)
 - Changed CO2 concentration used in RRTMG to be modifiable in geoschem_config.yml
 - Changed water vapor used in RRTMG to match to tracer field at all altitudes
+- Look for fullchem and TransportTracers restarts in the `GEOSCHEM_RESTARTS/GC_14.3.0` folder
```

Comment:
Update to only include fullchem since TransportTracers will be GC_14.2.0

Reply:
Now fixed in commit 8adbf0096